docs/source/catalog.rst

```diff
@@ -136,11 +131,88 @@ returned data.  Each data source has several attributes:
 
 - ``parameters``: A dictionary of data source parameters.  See below for more details.
 
-Parameters allow the user to customize the data returned by a data source.  Most often, parameters are used to filter
-or reduce the data in specific ways defined by the catalog author.  The parameters defined for a given data source are
-available for use in template strings, which can be used to alter the arguments provided to the driver.  For example,
-a data source might accept a "postal_code" argument which is used to alter a database query, or select a particular
-group within a file.  Users set parameters with keyword arguments to the ``get()`` method on the catalog object.
+
+Templating
+''''''''''
+
+Intake catalog files support Jinja2 templating for driver arguments. Any occurrence of
+a substring like ``{{field}}`` will be replaced by the value of the user parameters with
```

Comment:
Do we normally format that with spaces inside the double brackets?

Reply:
It makes no difference to jinja - I think it's fine if people don't worry about spacing here.